### PR TITLE
chore: add default approved org requests to dev data

### PR DESCRIPTION
**Pull request by dleard**
adds default approved org requests for test user & cypresstestreporter for all orgs (except SFO operator in case we want to look at the request process)

### CODE REVIEW

File: schema/data/dev/ciip_user_organisation.sql [hunk 1]
```diff
@@ -13,22 +8,27 @@ begin;
...
   with rows as (
   insert into ggircs_portal.ciip_user_organisation(id, user_id, organisation_id, status)
   overriding system value
   values
-  (1, 3, 8, 'approved'),
-  (2, 3, 7, 'approved'),
-  (3, 6, 1, 'approved'),
-  (4, 6, 7, 'approved'),
-  (5, 6, 8, 'approved')
+  (1, 3, 2, 'approved'),
```
Comment (matthieu-foucault) [inline schema/data/dev/ciip_user_organisation.sql:16]: Only `organisation_id` 1 is guaranteed to exist, as there is only one org when deploying the load testing data

Comment (matthieu-foucault) [reply schema/data/dev/ciip_user_organisation.sql:16]: I'll update the load testing cron job so that it doesn't call this script

Comment (dleard) [reply schema/data/dev/ciip_user_organisation.sql:16]: So no changes needed here? I was looking into conditionally adding values to the ciip_user_organisation table based on the count of organisations in the organisation table with a do block here.

Comment (matthieu-foucault) [reply schema/data/dev/ciip_user_organisation.sql:16]: I was thinking about this too. Having a different file and calling the script with a different parameter would be cleaner than having a conditional depending on a previous script's behavior

Comment (matthieu-foucault) [reply schema/data/dev/ciip_user_organisation.sql:16]: So no, no change needed here

**Review by matthieu-foucault — APPROVED** [review]
Looks good. Just needs a rebase

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**